src/backend/backend.c

```diff
@@ -479,8 +479,9 @@ void paint_all_new(session_t *ps, struct managed_win *t, bool ignore_damage) {
 
 				// Only animate opacity here if we are resizing
 				// a transparent window
+				double animation_progress_opacity = w->animation_progress > ps->o.animation_opacity_min ? w->animation_progress : ps->o.animation_opacity_min;
```

Comment:
Please change the conditional to `w->animation_progress < ps->o.animation_opacity_min ? ps->o.animation_opacity_min : w->animation_progress` in order to not cause bizarre opacity if `w->animation_progress` happens to be NaN.